kernel/kernel-authcodeflowproxy-api/src/main/java/io/mosip/kernel/authcodeflowproxy/api/service/validator/ValidateTokenHelper.java

```diff
@@ -111,15 +111,7 @@ public ImmutablePair<Boolean, AuthErrorCode> isTokenValid(DecodedJWT decodedJWT)
 		}
 
 		// Third, signature validation.
-		try {
-			String tokenAlgo = decodedJWT.getAlgorithm();
-			Algorithm algorithm = getVerificationAlgorithm(tokenAlgo, publicKey);
-			algorithm.verify(decodedJWT);
-		} catch (SignatureVerificationException signatureException) {
-			LOGGER.error("Signature validation failed, Throwing Authentication Exception. UserName: " + userName,
-					signatureException);
-			return ImmutablePair.of(Boolean.FALSE, AuthErrorCode.UNAUTHORIZED);
-		}
+		verifyJWTSignagure(decodedJWT);
```

Comment:
the return type is not handled here, should we not check and return when the JWT validation signature fails ?

Reply:
Yes Vishwa, we need to handle it. I have corrected it. Please check now. Thanks.